client/app/templates/components/game-client.hbs

```diff
@@ -0,0 +1,33 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-12 console">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span class="glyphicon glyphicon-comment"></span> Client
+        </div>
+        <div class="panel-body">
+          <ul class="chat">
+            {{#each messages as |msg|}}
+              <li class="clearfix">
+                <div class="chat-body clearfix">
+                  {{#if msg.server}}
```

Comment:
That breaks what I did

```html
{{#if msg.server}}
   <p class="response">{{msg.server.text}}</p>
{{else}}
   <p>&gt; {{msg.client.text}}</p>
{{/if}}
```

notice the `&gt;` in the else, hence why I separated them otherwise I could have done that. I want it to show your command then under it the response form the server so keeping that arrow was important to me much like a prompt does in a terminal.